corehq/apps/users/static/users/js/enterprise_users.js

```diff
@@ -44,24 +45,40 @@ hqDefine("users/js/enterprise_users", [
         self.toggleLoginAsUsers = function (webUser) {
             webUser.expanded(!webUser.expanded());
             _.each(self.users(), function (user) {
-                console.log(user);
-                self.showHideDeactivated(webUser);
-                if (user.loginAsUser === webUser.username && user.is_active) {
-                    user.visible(webUser.expanded() && !self.showDeactivated());
+                if (user.loginAsUser === webUser.username) {
```

Comment:
could condense this block to `user.visible(webUser.expanded() && user.is_active !== self.showDeactivated())`